docs/topics/api/overview.rst

```diff
@@ -327,41 +296,36 @@ API Versions
 ------------
 
 ~~~~~~~~~~~~~~
-Default v4 API
+Default v5 API
 ~~~~~~~~~~~~~~
 
-All documentation here, unless otherwise specified, refers to the default `v4` APIs,
+All documentation here, unless otherwise specified, refers to the default `v5` APIs,
 which are considered stable.
 The request and responses are *NOT* frozen though, and can change at any time,
 depending on the requirements of addons-frontend (the primary consumer).
 
 
 ~~~~~~~~~~~~~
-Frozen v3 API
+Frozen v4 API
 ~~~~~~~~~~~~~
 
 Any consumer of the APIs that requires more stablity may consider using
-the `v3` API instead, which is frozen.  No new API endpoints (so no new features)
-will be added to `v3` and we aim to make no breaking changes.
+the `v4` API instead, which is frozen.  No new API endpoints (so no new features)
+will be added to `v4` and we aim to make no breaking changes.
 Despite the aim, we can't guarantee 100% stability.
-The `v3` API will be maintained for as long as Firefox ESR60 is supported by Mozilla,
-i.e. at least October 23rd 2019.
 
-The documentation for `v3` can be accessed at: :ref:`v3-api-index`
+The documentation for `v4` can be accessed at: :ref:`v4-api-index`
 
 
- .. _api-experimental-v5:
+~~~~~~~~~~~~~~~~~
+Deprecated v3 API
+~~~~~~~~~~~~~~~~~
 
-~~~~~~~~~~~~~~~~~~~
-Experimental v5 API
-~~~~~~~~~~~~~~~~~~~
+The `v3` is now deprecated.  If you are using this API you should switch to `v4`,
+which is now frozen.
+The `v3` API will be maintained and available until at least 31st December 2021.
```

Comment:
This is the date we talked about internally, but it maybe should be a more conservative date just after the 78ESR support period ends e.g. 10th October 2021 so we're not already committed to keep it around longer than necessary for Firefox ESR.  Later, pushing the date back to the end of the year is a lot less impactful for 3rd parties than the other way around.

Reply:
I like 31st December, it's easy to remember :)